platform/android/src/style/sources/sources.cpp

```diff
@@ -20,6 +20,11 @@ namespace {
 
     Source* initializeSourcePeer(mbgl::style::Source& coreSource) {
         Source* source;
+        if (coreSource.peer.has_value()) {
+            source = mbgl::util::any_cast<std::unique_ptr<Source>>(&coreSource.peer)->get();
+            return source;
```

Comment:
Nit: follow the existing convention here -- change the subsequent `if` to `else if` and omit the early return -- or convert all the branches to early return and eliminate the `source` temporary variably entirely.